ui/components/multichain/asset-picker-amount/asset-picker-modal/Asset.tsx

```diff
@@ -22,6 +23,7 @@ export default function Asset({
   symbol,
   decimalTokenAmount,
   tooltipText,
+  showPercentage,
```

Comment:
We can remove this prop since this component is exclusive to the `AssetPickerModal`